src/docfx/build/xref/ExternalXrefMapLoader.cs

```diff
@@ -82,15 +82,15 @@ private static void LoadZipFile(Dictionary<string, Lazy<ExternalXrefSpec>> resul
                             var xrefMap = YamlUtility.Deserialize<XrefMapModel>(sr.ReadToEnd(), new FilePath(url));
                             foreach (var spec in xrefMap.References)
                             {
-                                result.TryAdd(spec.Uid, new Lazy<ExternalXrefSpec>(() => spec));
+                                result[spec.Uid] = new Lazy<ExternalXrefSpec>(() => spec);
```

Comment:
this means the latter one has higher priority, does it apply to all or only works in zip files?

Reply:
for the other parts, we are still using TryAdd

Reply:
It only applies for zip files as v2 does. And yes, the behavior is inconsistent in v2

Reply:
great to know